docs/upgrading/moving-22-30-basic.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Upgrading: Starting fresh (basic)"
+---
+Setting up a fresh Bolt 3.0, moving content
+=======================================
+
+This page describes the process of setting up a fresh Bolt 3.0 install, and moving over all of your existing configuration, data, themes and uploaded media.
+
+If you're running a custom Bolt 2.2 install, see [this](moving-22-30-basic).
```

Comment:
> If you're running a custom Bolt 2.2 install, see

It would depend on how customised it is… wonder if this would work better:

> Depending on how heavily customised your Bolt install is, you might need to take additional steps into account. The advanced upgrade guide can be found [here](moving-22-30-basic).